### DISCUSSION TITLE
PPA repositories

### DISCUSSION THREAD

**Discussion by yury-komarov-contextlabs** [renovatebot/renovate#25497] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.36.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello everybody!

I'm interesting, is it possible to track changes in launchpad repos?

Thank you!

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Probably yes, using the `repology` datasource

**Reply by yury-komarov-contextlabs** [reply]
but i didn't find launchpad repos in repology. or u mean that first need to add PPA to repology?

**Reply by rarkins** [reply, marked as answer]
I assumed they might be supported there. If not then unfortunately I have no further suggestion

**Reply by yury-komarov-contextlabs** [reply]
got it, thx
